TypeCobol/Parser.cs

```diff
@@ -149,12 +150,14 @@ public IEnumerable<CodeElement> CodeElements
 
 
 
-		public static Parser Parse(string path, DocumentFormat format, bool autoRemarks = false, IList<string> copies = null) {
+		public static Parser Parse(string path, DocumentFormat format, bool autoRemarks = false, IList<string> copies = null, bool checkProgramName = true) {
 			var parser = new Parser();
             var typeCobolOption = new TypeCobolOptions() { ExecToStep = ExecutionStep.Generate };
 #if EUROINFO_RULES
 		    typeCobolOption.AutoRemarksEnable = autoRemarks;
 #endif
+            if (!checkProgramName) typeCobolOption.CheckProgramName = new TypeCobolCheckOption(false);
```

Comment:
Same remark: equivalent to `typeCobolOption.CheckProgramName = new TypeCobolCheckOption(checkProgramName);`.